src/assets/drizzle/styles/utils/link.css

```diff
@@ -0,0 +1,4 @@
+.u-hiddenLink:not(:hover, :focus, :active) {
```

Comment:
Is the desire here to make the link not appear as a link except for when hovering?

If so, then 👍, but I'd change the name to `u-linkHidden` to better align with he SUIT link helpers that we already have in use: https://github.com/suitcss/utils-link/#available-classes